nxdrive/commandline.py

```diff
@@ -453,6 +454,10 @@ def _configure_logger(self, command: str, options: Namespace) -> None:
     def uninstall(self, options: Namespace = None) -> None:
         AbstractOSIntegration.get(None).uninstall()
 
+    def normalize_path(self, path: str) -> Path:
```

Comment:
This method could be static, or even a top-level function?

Reply:
Yes, and perhaps should we use it on `Options` directly. Like that we could expand variables from others sourcs (config.ini, server config).